diff --git a/base/BUILD.gn b/base/BUILD.gn
index 2beb9d2c713f86a1081644045a0ed720316a677f..ba7b928fa782d4fb1a7320db4a1f0d63aada3103 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4480,6 +4480,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4515,6 +4516,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
