diff --git a/chrome/elevation_service/BUILD.gn b/chrome/elevation_service/BUILD.gn
index 6ae8694531550df2835f3afbe5a699403f7495fb..a4328f979fa61c945cdb48f5728f1fff1a8b311f 100644
--- a/chrome/elevation_service/BUILD.gn
+++ b/chrome/elevation_service/BUILD.gn
@@ -9,7 +9,7 @@ import("//chrome/process_version_rc_template.gni")
 import("//testing/test.gni")
 
 midl("elevation_service_idl") {
-  sources = [ "elevation_service_idl.idl" ]
+  sources = [ "//brave/chromium_src/chrome/elevation_service/elevation_service_idl.idl" ]
 
   writes_tlb = true
 }
@@ -87,6 +87,7 @@ source_set("lib") {
   deps = [
     "//base",
     "//chrome/install_static:install_static_util",
+    "//brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper:common",
     "//components/crx_file:crx_file",
     "//third_party/zlib/google:zip",
   ]
