diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 5cdd437fff6969147c719245988b7a0c69dd158b..23af7e43c1c59012694d3f6c474b322323a3936c 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -820,6 +820,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1114,6 +1115,7 @@ void AutocompleteController::SortCullAndAnnotateResult(
   internal_result_.SortAndCull(input_, template_url_service_,
                                triggered_feature_service_,
                                default_match_to_preserve);
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
 
 #if DCHECK_IS_ON()
   internal_result_.Validate();
