diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index cf69488dbce25c06f818e28d1740a124e80a7649..8175577d27c4fd44169ac00d5aefe7ba8c98dda1 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4618,6 +4618,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
