diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 1a37b5da3bf9ae07cd6726ee7d3210e96d112ec5..0e99b178783355190ba9084076a0bd4bd26028e7 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -899,6 +899,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
