diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 1cfb8ade361f23a9865b41ae00c4478d103dcbd4..9325a3c585d30f34b6900a310bd723375ff10af9 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2292,6 +2292,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
