diff --git a/ios/chrome/app/main_controller.mm b/ios/chrome/app/main_controller.mm
index 0dd47928bdbd1960f1238baf702945d6c9d6d756..ce06edc5bca566f3f6f0a0b6efc191d577edb09b 100644
--- a/ios/chrome/app/main_controller.mm
+++ b/ios/chrome/app/main_controller.mm
@@ -140,6 +140,9 @@
 #import "ios/chrome/browser/credential_provider/model/credential_provider_service_factory.h"
 #import "ios/chrome/browser/credential_provider/model/credential_provider_support.h"
 #import "ios/chrome/browser/credential_provider/model/credential_provider_util.h"
+#endif
+
+#if BUILDFLAG(FAST_APP_TERMINATE_ENABLED)
 #import "ios/chrome/browser/sessions/session_restoration_service.h"
 #import "ios/chrome/browser/sessions/session_restoration_service_factory.h"
 #endif
