diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 8b0fb967bf7bd58d0220968fbe0aceb5277b30ac..be623ee117dabe35e69fd19a6ea4c8e226d50fa9 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -753,8 +753,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
