diff --git a/third_party/rust/winapi/v0_3/BUILD.gn b/third_party/rust/winapi/v0_3/BUILD.gn
index 81e6c1169640b03941338b152be636994a1b934c..5c078ba997d39e1ad0d4f551e1d6be7683a0c231 100644
--- a/third_party/rust/winapi/v0_3/BUILD.gn
+++ b/third_party/rust/winapi/v0_3/BUILD.gn
@@ -444,6 +444,7 @@ cargo_crate("lib") {
     "winerror",
     "winnt",
   ]
+  features += [ "basetsd", "cfg", "cfgmgr32", "devpropdef", "excpt", "handleapi", "ktmtypes", "libloaderapi", "limits", "ntdef", "ntstatus", "processthreadsapi", "reason", "rpcndr", "timezoneapi", "vadefs", "vcruntime", "wincontypes", "wingdi", "windef", "winreg", "winuser", "wtypesbase" ]
   build_root = "crate/build.rs"
   build_sources = [ "crate/build.rs" ]
 
