diff --git a/third_party/rust/syn/v1/BUILD.gn b/third_party/rust/syn/v1/BUILD.gn
index 61440b7ca68a27f00e217fc3a35cd822374fed7a..fe5dc79aecd4a331a18c5530c7f19185bdc59381 100644
--- a/third_party/rust/syn/v1/BUILD.gn
+++ b/third_party/rust/syn/v1/BUILD.gn
@@ -134,6 +134,7 @@ cargo_crate("lib") {
     "visit",
     "visit-mut",
   ]
+  features += [ "fold" ]
   build_root = "crate/build.rs"
   build_sources = [ "crate/build.rs" ]
 
