diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 064aeed7f0c29b9ea70d4f1d3825ef2dcc43f032..8bdb82b13fa60aa1f2ac1752aa13d578fddf459a 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -458,6 +458,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
