diff --git a/third_party/rust/rand/v0_8/BUILD.gn b/third_party/rust/rand/v0_8/BUILD.gn
index bea7ba93b78aa99ad2c936c06f982323d3689a4f..dcdc30221a6032e3f1d5d6acbf663125f24396f6 100644
--- a/third_party/rust/rand/v0_8/BUILD.gn
+++ b/third_party/rust/rand/v0_8/BUILD.gn
@@ -72,6 +72,7 @@ cargo_crate("lib") {
     "std",
     "std_rng",
   ]
+  features += [ "small_rng" ]
 
   # Only for usage from third-party crates. Add the crate to
   # third_party.toml to use it from first-party code.
