diff --git a/BUILD.gn b/BUILD.gn
index 423625d038dbc13830c6239ef3860f54451ad702..b9ca786f37a09f39d50c981863dd373cf869a503 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -739,6 +739,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3015,6 +3016,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
