diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index fa0e6edbada3aff66763a6ed0a575637f98c99b5..c802c8e1a1e0e643856c7b8fadb981f5fefc8b46 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2989,6 +2989,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
